Alvaro Gaming/PythonGaming_p3.py

```diff
@@ -21,6 +27,8 @@
 # Setting up other variables that will be used in program
 SCREEN_WIDTH = 800
 SCREEN_HEIGHT = 600
+BALL_BOUNDX = 750
+BALL_BOUNDY = 510
```

Comment:
I think these should be stored in the ball class, since that's the only place that cares about it and needs to use it.